### PR TITLE
Suggested revisions

### PR DESCRIPTION
**Pull request by nfahlgren**
I added suggested revisions to the source markdown documents. There are a few places I added some additional markdown code, but I'm less familiar with R-markdown and pandocs, so might need to check that they work (super and subscripts in particular).

### CODE REVIEW

**Comment by nfahlgren** [comment]
I saw the legend for Figure 1 but not the other figures

File: content/environmental.Rmd [hunk 1]
```diff
@@ -92,23 +92,23 @@ cowplot::plot_grid(plots[[1]], plots[[2]], ncol = 1)
...
 These are even higher resolution time series.
 
-The files for each season are approximately 40GB and, like sensor data, are provided using methods such as Globus transfer or the Clowder API (see "How To Access Data" in the [documentation](https://docs.terraref.org/user-manual/how-to-access-data)). On Globus, they are in the `/ua-mac/Level_1/envlog_netcdf/`.
+The files for each season are approximately 40 GB and, like sensor data, are provided using methods such as Globus transfer or the Clowder API (see "How To Access Data" in the [documentation](https://docs.terraref.org/user-manual/how-to-access-data)). On Globus, they are in the `/ua-mac/Level_1/envlog_netcdf/` directory.
 
 The data are described in the TERRA REF documentation under the environmental data section of the [data products chapter](https://docs.terraref.org/user-manual/data-products/environmental-conditions).
 
-The primary reason that these files are so large is that they contain 
+The primary reason that these files are so large is that they contain
 Here, I am assuming that these have been downloaded to a folder named `~/data/ua-mac/Level_1/envlog_netcdf/`
```
Comment (nfahlgren) [inline content/environmental.Rmd]: These sentences looked truncated.

File: content/environmental.Rmd [hunk 2]
```diff
@@ -92,23 +92,23 @@ cowplot::plot_grid(plots[[1]], plots[[2]], ncol = 1)
...
 
 These are even higher resolution time series.
 
-The files for each season are approximately 40GB and, like sensor data, are provided using methods such as Globus transfer or the Clowder API (see "How To Access Data" in the [documentation](https://docs.terraref.org/user-manual/how-to-access-data)). On Globus, they are in the `/ua-mac/Level_1/envlog_netcdf/`.
+The files for each season are approximately 40 GB and, like sensor data, are provided using methods such as Globus transfer or the Clowder API (see "How To Access Data" in the [documentation](https://docs.terraref.org/user-manual/how-to-access-data)). On Globus, they are in the `/ua-mac/Level_1/envlog_netcdf/` directory.
 
 The data are described in the TERRA REF documentation under the environmental data section of the [data products chapter](https://docs.terraref.org/user-manual/data-products/environmental-conditions).
 
-The primary reason that these files are so large is that they contain 
+The primary reason that these files are so large is that they contain
```
Comment (dlebauer) [inline content/environmental.Rmd]:
```suggestion
The primary reason that these files are so large is that they contain the downwelling spectral radiance measurements (350-800 nm) at five second intervals.
```

**Review by dlebauer — APPROVED** [review]
Thanks for the review and close attention to details!

**Pull request merged**